.gitignore

```diff
@@ -43,3 +43,10 @@ build/config/buildinfo.properties
 
 www-built
 www-ghdeploy
+/node_modules/
+/tmp/
+/scripts/kiwix-*.pem
+/scripts/set_secret_environment_variables.sh
```

Comment:
At the risk of stating the obvious, having secret content exposed on an open source repo seems bizarre / counter-productive :)